lib/createCookieEncryptor.js

```diff
@@ -73,12 +73,15 @@ export const createCookieEncryptor = (opts) => {
         obj[name] = value
         return obj
       }, {})
-      if (!cookieObj.hasOwnProperty(cookieName)) {
-        return missingCookieHandler(request, event)
+      const hasCookie = cookieObj.hasOwnProperty(cookieName)
+      if (!hasCookie) {
+        missingCookieHandler(request, event)
+      }
+      if (hasCookie) {
```

Comment:
Is there a reason this is two separate conditionals? Is there a case where both conditionals should run?

Reply:
good q!

Reply:
good catch